source/photo.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Пинк</title>
+  </head>
+  <body class="page__body">
+  	<header class="page-header">
+  		<nav class="main-nav">
+  			<button class="main-nav__toggle" type="button">
+  			<span class="visually-hidden">Открыть меню</span>
+      		</button>
+	    <div class="main-nav__wrapper">
+	      <ul class="main-nav__list site-list">
+	        <li class="site-list__item">
+	          <a href="index.html">
+	            Главная
+	          </a>
+	        </li>
+	        <li class="site-list__item site-list__item--active">
+	          <a>
+	            Фотографии
+	          </a>
+	        </li>
+	        <li class="site-list__item">
+	          <a href="form.html">
+	            Конкурс
+	          </a>
+	        </li>
+	        <li class="site-list__item">
+	          <a href="https://htmlacademy.ru">
+	            HTML Academy
+	          </a>
+	        </li>
+	      </ul>
+  			</div>
+    	</nav>
+  	</header>
+
+  	<main class="page-main">
+  		<div class="page-main__background">
+  			<p>Фото пользователей</p>
+  			<img  src="https://via.placeholder.com/320x462" alt="фон">
+  		</div>
+
+    <section class="reviews">
+      <p class="page-title">Взгляните на фотографии, которые выкладывают пользователи! Видите, как не хватает ваших?</p>
+      <div class="reviews__list">
+      	<div class="reviews__wrapper slider">
+	        <figure class="reviews__item">
+	          <img class="reviews__author-image" src="https://via.placeholder.com/320x137" alt="Фото Анны">
+	          <figcaption class="reviews__author-info">
+	            <span class="reviews__author-name">Анна</span>
+	            <span class="reviews__author-time">(5 минут назад)</span>
+	            <p class="reviews__text">Прыгать или нет? Напишите в комментарии свой совет и смотрите прямую трансляцию в перископе, задавайте свои вопросы!</p>
+	          <button class="like-button"></button>
```

Comment:
```suggestion
	          <button class="like-button" type="button"></button>
```
Если явно не указать `type`, то его значение по умолчанию равно `submit`.
В остальных карточках тоже исправьте.